app/sider/Dokumentasjon.tsx

```diff
@@ -0,0 +1,11 @@
+import HovedInnhold from '~/komponenter/hovedInnhold/HovedInnhold';
+import StegIndikator from '~/komponenter/stegindikator/StegIndikator';
+
+export default function DokumentasjonSide() {
+  return (
+    <HovedInnhold /*måHaBekreftetSamtykke*/>
```

Comment:
Har kommentert ut `måHaBekreftetSamtykke` inntil navigering er på plass. Hvis ikke vil vi navigeres rett til forsiden uten mulighet for å komme oss tilbake.

Reply:
Smuud ☀️